src/MahApps.Metro.Samples/MahApps.Metro.Demo/ExampleViews/DataGridExamples.xaml

```diff
@@ -177,9 +177,15 @@
                                         EditingElementStyle="{DynamicResource MahApps.Styles.CheckBox.DataGrid}"
                                         ElementStyle="{DynamicResource MahApps.Styles.CheckBox.DataGrid}"
                                         Header="Album Selected" />
-                <DataGridTextColumn Binding="{Binding Title}" Header="Title" />
-                <DataGridTextColumn Binding="{Binding Artist.Name}" Header="Artist" />
-                <DataGridTextColumn Binding="{Binding Genre.Name}" Header="Genre" />
+                <DataGridTextColumn Binding="{Binding Title}"
+                                    EditingElementStyle="{StaticResource MahApps.Styles.TextBox.DataGrid.Editing}"
```

Comment:
Do you think there is a way to autoapply this (with `AttachedProperties` or a `Behaviour`) for those who use autogenerated columns? If not we should provide a code snipped for the needed steps in code behind IMO.

Reply:
@timunie done